src/app/visual/action-button/action-button-visual.component.html

```diff
@@ -38,7 +38,7 @@ <h3>Basic action button</h3>
 </div>
 
 <h3>Action buttons inside sky-action-button-container</h3>
-<div *skyThemeIf="'modern'"
+<div
```

Comment:
☝️ Now that we support center/left alignment in default theme, this if check is no longer needed.